.gitignore

```diff
@@ -16,5 +16,6 @@ containers/datalab/content/license.txt
 containers/gateway/Dockerfile
 containers/gateway/content/license.txt
 containers/base/pydatalab
+test/node_modules
 node_modules
-test/ui/broken
+test/ui/broken
```

Comment:
I'd like to keep in the same paragraph, because it's related to the `ui/golden` bit.